### ISSUE TITLE
Searching dates

### ISSUE THREAD

**Issue by ghost** [mmanos/laravel-search#5]
Hi,

This is a question regarding ranges so mark it please.

My question is that is there support for date search? I`m building a portal and i need to implement ranged search with dates, for example i need to search the posts which were published in the past month.

Thanks!

**Comment by dmyers** [comment]
Yeah it's bit complicated to use the advanced query callback. Essentially you can get access to the driver's client and then use exposed methods to do advanced queries, but you will be tied to that driver for that implementation. It's a bit complicated and requires you to really dive into the package and understand the internals.

Here's the example from the readme. See how it is altering the `$query` array? That array is passed to the driver, in this case Elastic Search.
```php
$results = Search::index('posts')->select('id', 'created_at')
    ->search('content', 'fox')
    ->addCallback(function ($query) {
        // Adjust pagination for an elasticsearch query array.
        $query['from'] = 0;
        $query['size'] = 20;
        return $query;
    }, 'elasticsearch')
    ->get();
```
